pkg/pligos/config.go

```diff
@@ -38,18 +48,24 @@ func FindContext(name string, contexts []Context) (Context, bool) {
 	return Context{}, false
 }
 
+type Dependency struct {
+	Pligos  string `yaml:"pligos" filepath:"resolve"`
```

Comment:
What does Dependency.Pligos represent? It feels to me that it doesnt reflect a semantic meaning, otherwise I'd not have to ask.

Reply:
path to pligos config